CHANGELOG.md

```diff
@@ -12,10 +12,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [Core] Omit filtered out pickles from html report ([react-components/#273](https://github.com/cucumber/react-components/pull/273) David J. Goss)
 
 ## [Unreleased]
+### Changed
+- Always color Maven output in GitHub workflows with -Dstyle.color=always ([#2616](https://github.com/cucumber/cucumber-jvm/pull/2616) Tim te Beek)
```

Comment:
Cheers! Looks good, but can you remove this entry? Nothing about the artifacts we distribute as Cucumber changed so we don't have to inform people about them.

Reply:
Yeah sure; I'd only added it after seeing the note in the pull request template; I'd thought to leave in the attribution that was missing. Up to you if you want that kept or reverted.